inference-engine/src/transformations/include/transformations/rt_info/old_api_map_attribute.hpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief Defines old API map attribute
+ * @file old_api_map_attribute.hpp
+ */
+
+#pragma once
+
+#include <assert.h>
+
+#include <functional>
+#include <memory>
+#include <ngraph/attribute_visitor.hpp>
+#include <ngraph/node.hpp>
+#include <ngraph/variant.hpp>
+#include <openvino/core/rtti.hpp>
+#include <set>
+#include <string>
+#include <transformations_visibility.hpp>
+#include <utility>
+
+namespace ov {
+
+class OldApiMap;
+/**
+ * @ingroup ie_runtime_attr_api
+ * @brief OldApiMapAttr class stores the value of OldApiMap class.
+ */
+class TRANSFORMATIONS_API OldApiMapAttr {
+private:
+    std::vector<uint64_t> m_order;
+    ngraph::element::Type m_type;
+
+public:
+    friend class OldApiMap;
+
+    /**
+     * A default constructor
+     */
+    OldApiMapAttr() = default;
+
+    /**
+     * @brief      Constructs a new OldApiMapAttr object.
+     * @param[in]  order  Transpose order.
+     * @param[in]  type  Legacy type.
+     */
+    explicit OldApiMapAttr(std::vector<uint64_t> order, ngraph::element::Type type) {
+        m_order = std::move(order);
+        m_type = type;
+    }
+
+    /**
+     * @brief Returns the transpose order that should be used for obtain a node with old API layout.
+     * @return transpose order.
+     */
+    std::vector<uint64_t> get_order() const {
+        return m_order;
+    }
+
+    /**
+     * @brief Returns the legacy type of the node.
+     * @return legacy type.
+     */
+    ngraph::element::Type get_type() const {
+        return m_type;
+    }
+};
+
```

Comment:
I would suggest to add helpers like this:
`add_old_api_map`
`has_old_api_map`
`set_old_api_map`

Reply:
Done.